### PR TITLE
#442: Added Cancel Button to pick a size screen

### PR DESCRIPTION
**Pull request by ahmed-shehata**
#442: Added Cancel Button to pick a size screen

### CODE REVIEW

**Comment by ahmed-shehata** [comment]
👍

**Comment by ahmed-shehata** [comment]
👍

File: AtlasUI/AtlasUI/SizeSelectionViewController.swift [hunk 1]
```diff
@@ -79,9 +79,29 @@ final class SizeSelectionViewController: UIViewController, CheckoutProviderType
         checkout.client.article(forSKU: sku) { [weak self] result in
             guard let strongSelf = self, article = result.process() else { return }
             strongSelf.displaySizes(forArticle: article)
+            strongSelf.showCancelButton()
```
Comment (hani-ibrahim) [inline AtlasUI/AtlasUI/SizeSelectionViewController.swift]: hide loader too?

**Comment by hani-ibrahim** [comment]
👍

**Comment by ahmed-shehata** [comment]
👍

**Pull request merged**